package.json

```diff
@@ -19,7 +19,7 @@
   "license": "ISC",
   "dependencies": {
     "cids": "^0.8.3",
-    "multicodec": "^1.0.4",
+    "multicodec": "^2.1.0",
```

Comment:
This update causes a bunch of test failures. It seems that the format changed causing breakage in most if not all of the encode-decode functions. Not sure what to do here.